test/controllers/PaypalControllerSpec.scala

```diff
@@ -168,25 +168,27 @@ class PaypalControllerSpec extends PlaySpec
 
   "Paypal Controller" should {
 
-    "generate correct redirect URL for support's successful PayPal payments" in {
+    "return the email address when called from support (and accept type is application/json)" in {
```

Comment:
This test is nice - very clear. Maybe we could we add one to test when it's called from contribute?

Reply:
This test already exists below, in the `"generate correct redirect URL for successful PayPal payments"` I'll make the name clearer